spec/classes.md

```diff
@@ -2873,9 +2873,16 @@ the underlying `TextWriter` for the output device is created. But if the applica
 
 ### Automatically implemented properties
 
-An automatically implemented property (or ***auto-property*** for short), is a non-abstract non-extern property with semicolon-only accessor bodies. Auto-properties must have a get accessor and can optionally have a set accessor.
+An automatically implemented property (or ***auto-property*** for short), is a non-abstract non-extern property with at least:
 
-When a property is specified as an automatically implemented property, a hidden backing field is automatically available for the property, and the accessors are implemented to read from and write to that backing field. If the auto-property has no set accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto-property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the readonly backing field of the property.
+1. one semicolon-only body
+2. usage of the `field` contextual keyword ([Keywords](lexical-structure.md#keywords)) within the `property_body` of the `property`. The `field` identifier is only considered the `field` contextual when there is no symbol named `field` in scope at that location. For the purposes of simplicity, from this point on any usages of `field` in the specification description will assume there is no `field` symbol in scope at that location.
+
+Auto-properties must have a get accessor and can optionally have a set accessor. Because an expression-bodied property defines a get accessor, if the expression-bodied property refers to `field`, it satisfies the requirements and is considered an auto-property.
+
+When a property is specified as an auto-property, a hidden, unnamed, backing field is automatically available for the property, and any semicolon-only accessors are implemented to read from and write to that backing field. The hidden backing field is in scope in all accessors and can be referenced directly using `field`.  Because the field is unnamed, it cannot be used in a `nameof` expression.
+
+ If the auto-property has no set accessor, and does not use `field`, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto-property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the readonly backing field of the property.
```

Comment:
That doesn't seem obvious. Can't I have an auto-incrementing get-only property?

Reply:
this was tweaked to separate out the concepts of 'readonly' (which only occurs for a `get;`-only autoprop) and 'constructor can assign directly to field'.  Can you recheck?